migrations/versions/c88ac0c951d2_.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
```

Comment:
Can we give this one a name? Looks like "add title to conversation"? OR... should the changes to the conversation that happen in this migration just be rolled into [add_assistant_to_conversation.py](https://github.com/RedHatInsights/tangerine-backend/pull/185/files#diff-14fccd3f6d846d2d4a2d341376c19f0ae29abd57961fb605d51665bf9068fc92) ?